### ISSUE TITLE
error TS2305: Module '"typy"' has no exported member 't'

### ISSUE THREAD

**Issue by burner** [flexdinesh/typy#28]
when importing type  
```TS
import { t } from "typy";
```

with typescript 3.4.3

I get the following error
```
error TS2305: Module '"typy"' has no exported member 't'.

6 import { t } from "typy";
```

looking at the index.d.ts this looks right.
t is actually a member of Type.
But creating a new Type and than calling `t` on it looks really clunky.

Not sure what to do here.

**Comment by flexdinesh** [comment]
@analizapandac do you have any idea what this might be?

**Comment by analizapandac** [comment]
@flexdinesh As @burner mentioned, `t` is a member of `Typy` however it's also defined as the default export. 

I'm not sure yet how to fix this one, I will check this out. For now, since you're using ES6 import style, you can use the default `t`.

```js
import t from "typy"
```

**Comment by burner** [comment]
I tried add 

```js
declare module "typy" {
	export function t(obj: any, nestedKeys?: string): Typy
```

to my index.d.ts

and Typescript is happy again

**Comment by analizapandac** [comment]
@burner You're right. Replacing `typy` with `t` instead solved the issue.
```js
export function t(input: any, objectPath?: string): Typy
export default t;
```

**Comment by burner** [comment]
"Schema, addCustomTypes"  should be exported differently then.

**Comment by analizapandac** [comment]
Yup, they should be included in the type definitions file as well since right now, both of these are not declared there.

**Comment by burner** [comment]
did we miss any other types?
can you create a new PR or should I?

**Comment by analizapandac** [comment]
I think just these 3 needs to be changed. You can create the PR then we'll review it :)

**Comment by flexdinesh** [comment]
Changes published in `v3.2.2`
